### PR TITLE
xilin_wang/hw1

### PR DESCRIPTION
**Pull request by Wxl19980214**
* Created mavenProject folder and a blank project
* hw1.md included

### CODE REVIEW

File: MavenProject/maven-learn/.gitignore [hunk 1]
```diff
@@ -0,0 +1,38 @@
+target/
```
Comment (CharlesCCC) [inline MavenProject/maven-learn/.gitignore:1]: actually there is no need to add another gitignore.

Comment (Wxl19980214) [reply MavenProject/maven-learn/.gitignore:1]: No idea why it has the gitignore there, i will delete it.

**Comment by Chingpo-Lin** [comment]
nice work

**Review by Hongming98 — APPROVED** [review]
Good Job!

**Pull request merged**